### ISSUE TITLE
Executing callhome_system_inventory_configure() and callhome_system_inventory_send_now() throws UcsOperationError

### ISSUE THREAD

**Issue by tanmayx** [CiscoUcs/ucsm_apis#29]
ucsm_apis version: Latest code
Python version: 2.7.12
Operating System: Win 10

**Description -**
Trying to execute below APIs
_callhome_system_inventory_configure()_
_callhome_system_inventory_send_now()_ 
_callhome_anonymous_reporting_on()_
_callhome_anonymous_reporting_off()_

**What I Did**
Executed the below code -
```
callhome_system_inventory_configure(handle=handle)
callhome_system_inventory_send_now(handle=handle)
callhome_anonymous_reporting_on(handle=handle)
callhome_anonymous_reporting_off(handle=handle)
```

Observed below error -

```
Traceback (most recent call last):
  File "c:\python27\lib\site-packages\nose\case.py", line 197, in runTest
    self.test(*self.arg)
  File "C:\Tanmay\PythonSDK\pysdkframework\TestSuite_Ucsm\Nosetests\admin\test_callhome.py", line 687, in test_callhome_system_inventory_configure_default
    obj = callhome_system_inventory_configure(handle=handle)
  File "c:\python27\lib\site-packages\ucsm_apis-0.9.0.0-py2.7.egg\ucsm_apis\admin\callhome.py", line 692, in callhome_system_inventory_configure
    "Callhome system inventory '%s' does not exist." % dn)
UcsOperationError: callhome_system_inventory_configure failed, error: Callhome system inventory 'call-homeperiodicsysteminventory' does not exist.
```
```
Traceback (most recent call last):
  File "c:\python27\lib\site-packages\nose\case.py", line 197, in runTest
    self.test(*self.arg)
  File "C:\Tanmay\PythonSDK\pysdkframework\TestSuite_Ucsm\Nosetests\admin\test_callhome.py", line 749, in test_callhome_anonymous_reporting_on
    user_acknowledged='yes')
  File "c:\python27\lib\site-packages\ucsm_apis-0.9.0.0-py2.7.egg\ucsm_apis\admin\callhome.py", line 765, in callhome_anonymous_reporting_on
    "Callhome Anonymous Reporting '%s' does not exist." % dn)
UcsOperationError: callhome_anonymous_reporting_on failed, error: Callhome Anonymous Reporting 'call-homeanonymousreporting' does not exist.
```

In callhome.py, line 688, 730 it should be - `dn = _base_dn + "/periodicsysteminventory"`
line 761, 794 it should be - `dn = _base_dn + "/anonymousreporting"`

**Comment by ragupta-git** [comment]
Fixed in PR: Added exists api and few fixes for callhome apis #48

**Comment by tanmayx** [comment]
Verified. Marking as closed.
